src/analysis/normed_space/operator_norm.lean

```diff
@@ -642,6 +642,48 @@ begin
       ... ≤ ∥smul_right c f∥ * ∥x∥ : le_op_norm _ _ } },
 end
 
+/-- Given `c : c : E →L[𝕜] 𝕜`, `c.smul_rightₗ` is the linear map from `F` to `(E →L[𝕜] F)`
+sending `f` to `λ e, c e • f`. -/
+def smul_rightₗ (c : E →L[𝕜] 𝕜) : F →ₗ[𝕜] (E →L[𝕜] F) :=
+{ to_fun := c.smul_right,
+  map_add' := λ x y, by { ext e, simp [smul_add] },
+  map_smul' := λ a x, by { ext e, simp [smul_comm] } }
+
+/-- Given `c : c : E →L[𝕜] 𝕜`, `c.smul_rightL` is the continuous linear map from `F` to `(E →L[𝕜] F)`
+sending `f` to `λ e, c e • f`. -/
+def smul_rightL (c : E →L[𝕜] 𝕜) : F →L[𝕜] (E →L[𝕜] F) :=
+(c.smul_rightₗ : F →ₗ[𝕜] (E →L[𝕜] F)).mk_continuous _ (λ f, le_of_eq $ c.norm_smul_right_apply f)
+
+@[simp] lemma norm_smul_rightL_apply (c : E →L[𝕜] 𝕜) (f : F) :
+  ∥c.smul_rightL f∥ = ∥c∥ * ∥f∥ :=
+by simp [continuous_linear_map.smul_rightL, continuous_linear_map.smul_rightₗ]
+
+@[simp] lemma norm_smul_right (c : E →L[𝕜] 𝕜) (hF : 0 < vector_space.dim 𝕜 F) :
```

Comment:
```suggestion
@[simp] lemma norm_smul_rightL (c : E →L[𝕜] 𝕜) (hF : 0 < vector_space.dim 𝕜 F) :
```
Also, can you replace the dimension assumption with `nontrivial F`?

Reply:
Yes, it's a nice idea, thanks!